Content/build.fsx

```diff
@@ -16,17 +16,12 @@ let deployDir = Path.getFullName "./deploy"
 
 let npm args workingDir =
     let npmPath =
-        let npmPath = if Environment.isUnix then "npm" else "npm.cmd"
-
-        npmPath
-        |> ProcessUtils.tryFindFileOnPath
-        |> Option.defaultWith(fun () ->
-            let errorMsg =
-                npmPath +
-                " was not found in path. " +
-                "Please install it and make sure it's available from your path. " +
-                "See https://safe-stack.github.io/docs/quickstart/#install-pre-requisites for more info"
-            failwith errorMsg)
+        match ProcessUtils.tryFindFileOnPath "npm" with
+        | Some path -> path
+        | None ->
```

Comment:
Could this be simplified into `Option.defaultWith(fun () -> ...)`?

Reply:
I actually just changed it away from that because I thought this was a bit simpler!

Reply:
True. I find that the `Some x -> x | None -> exn` pattern is more easily encoded with `defaultWith` - you can see a little more easily what's happening IMHO. But not fussed.